### ISSUE TITLE
Bump action

### ISSUE THREAD

**Issue by v1gnesh** [zopencommunity/buildkiteport#33]
Hi @IgorTodorovskiIBM, does this look familiar to you?

```sh
exec> git config --global --add safe.directory /github/workspace
exec> git config --global user.name zosopentoolsmain
exec> git config --global user.email bump-action@github
exec> git remote set-url --push origin ***github.com/zopencommunity/buildkiteport.git
Checking buildkite-version
  Updatable to 3.82.1
401 Unauthorized
```

**Comment by IgorTodorovskiIBM** [comment]
I got several emails today regarding the workflows failing - all due to the same reason. Wondering if it's a github issue - Normally I get an email when a token expires

**Comment by v1gnesh** [comment]
Maybe the `BUMP_TOKEN` needs to be renewed?

**Comment by IgorTodorovskiIBM** [comment]
> Maybe the `BUMP_TOKEN` needs to be renewed?

Yeah, I forgot zosopentoolsmain owned that one! :) I renewed that earlier today. However, something else changed, the read/write option is now greyed out.


<img width="947" alt="Screenshot 2024-09-30 at 10 08 56 PM" src="https://github.com/user-attachments/assets/a53e917d-bbce-4787-961e-108aa293c592">

So I was getting this issue: https://github.com/zopencommunity/bashport/actions/runs/11112346664

Adding permissions to the workflow fixed it: https://github.com/zopencommunity/bashport/blob/main/.github/workflows/build_and_test.yml#L3-L6 I think I'll need to do another mass change to all the workflows
